client/webserver/site/src/js/markets.ts

```diff
@@ -2676,7 +2676,10 @@ class BalanceWidget {
       const balTmpl = Doc.parseTemplate(row)
       balTmpl.title.textContent = title
       balTmpl.bal.textContent = Doc.formatCoinValue(bal, ui)
-      if (icon) balTmpl.bal.append(icon)
+      if (icon) {
+        balTmpl.bal.append(icon)
+        tmpl.parentBal = balTmpl.bal
```

Comment:
I see you've added `parentBal?: PageElement` on `BalanceWidgetElement` that seems intended for this purpose, but is unused. Can we use that instead?

```Typescript
side.parentBal = balTmpl.bal
```
here, and
```Typescript
 if (side.parentBal) side.parentBal.textContent = Doc.formatCoinValue(balance.available, unitInfo)
```
in `updateParent`.

Reply:
Oh, ok. Done.